header.php

```diff
@@ -2,43 +2,21 @@
 <html <?php language_attributes(); ?>>
   <head>
     <meta charset="<?php bloginfo( 'charset' ); ?>" />
-    <meta name="viewport" content="width=device-width">        
-    </div>
+    <meta name="viewport" content="width=device-width">
     <title><?php wp_title(); ?></title>
     
-
-    <link href='https://fonts.googleapis.com/css?family=Poiret+One' rel='stylesheet' type='text/css'>
-    <link href="https://fonts.googleapis.com/css?family=Cabin+Sketch|Dancing+Script|David+Libre|Open+Sans+Condensed:300|Poiret+One" rel="stylesheet">
-    <link href="https://fonts.googleapis.com/css?family=Wire+One" rel="stylesheet">
+    <!--google fonts-->
     <link href="https://fonts.googleapis.com/css?family=Loved+by+the+King|Pontano+Sans|Wire+One" rel="stylesheet">
     <link href="https://fonts.googleapis.com/css?family=Over+the+Rainbow" rel="stylesheet">
-    <link href="https://fonts.googleapis.com/css?family=Open+Sans+Condensed:300" rel="stylesheet">
-    <link href="https://fonts.googleapis.com/css?family=Sacramento" rel="stylesheet">
-    <link href="https://fonts.googleapis.com/css?family=Scope+One" rel="stylesheet">
-    <link href="https://fonts.googleapis.com/css?family=Waiting+for+the+Sunrise" rel="stylesheet">
-    <link href="https://fonts.googleapis.com/css?family=Pompiere" rel="stylesheet">
-    <link href="https://fonts.googleapis.com/css?family=Zeyada" rel="stylesheet">
-
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
```

Comment:
Try and limit the amount of typefaces you're using. It puts a lot of load time onto the site.